### PR TITLE
mssql-linux labels and annotations

### PR DESCRIPTION
**Pull request by matthewdevenny**
#### What this PR does / why we need it:
This PR adds the ability to specify additional labels and annotations on the pod, service and deployment spec for the mssql-linux. Adding this in my use case allows firewall rules to be written that are based upon the pod labels.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md

### CODE REVIEW

**Comment by boxboatmatt** [comment]
@olemarkus any thing you want me to change for this PR?

File: stable/mssql-linux/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: SQL Server 2017 Linux Helm Chart
 name: mssql-linux
-version: 0.6.5
+version: 0.6.6
```
Comment (olemarkus) [inline stable/mssql-linux/Chart.yaml]: Adding features here, so this should be minor version bumped. As this chart is in stable, I would actually do a bump to 1.0.0

**Comment by olemarkus** [comment]
This isn't my chart, so I am not sure why I was asked to review. But looks good to me except for the chart version.

/ok-to-test

File: stable/mssql-linux/Chart.yaml (hunk 1 above)
Comment (matthewdevenny) [reply stable/mssql-linux/Chart.yaml]: Updated to 1.0.0

**Comment by boxboatmatt** [comment]
@thomasliddledba can you review this PR?

**Comment by thomasliddledba** [comment]
@boxboatmatt - Can you correct the version in the `Chart.yaml`, please?

**Comment by boxboatmatt** [comment]
@thomasliddledba what version would you like me to set it to?

**Comment by thomasliddledba** [comment]
Bump the chart version as a minor version.  Thanks

**Comment by boxboatmatt** [comment]
@thomasliddledba should be good to go

**Pull request merged**